CommunicationServer/Application/Entities/ChessTimer.cs

```diff
@@ -0,0 +1,93 @@
+using Application.Utils;
+using Domain.DTOs;
+using Rudzoft.ChessLib.Enums;
+
+namespace Application.Entities;
+
+public class ChessTimer
+{
+    private readonly PausableTimer _whiteTimer = new(1000.0);
+    private readonly PausableTimer _blackTimer = new(1000.0);
+    private readonly uint _timeControlSeconds;
+    private readonly uint _timeControlIncrement;
+    private readonly ValueWrapper<bool> _whitePlaying;
+    public double WhiteRemainingTimeMs { get; private set; }
+    public double BlackRemainingTimeMs { get; private set; }
+    
+    public delegate void EventHandler(object sender, EventArgs args, ResponseJoinedGameDto dto);
+    public event EventHandler ThrowEvent = delegate{};
```

Comment:
ThrowEvent is name suitable for throwing exceptions.

Reply:
I copied it from the internet where they used it for events. And the internet is always right